tide_core.install

```diff
@@ -1384,3 +1384,27 @@ function tide_core_update_8058() {
     }
   }
 }
+
+/**
+ * Changes allowed HTML list.
+ */
+function tide_core_update_8059() {
+  // Update allowed HTML list.
+  $config_factory = \Drupal::configFactory();
+  $filter_ids = [
+    'filter.format.admin_text',
+    'filter.format.rich_text',
+    'filter.format.summary_text',
+  ];
+  foreach ($filter_ids as $filter_id) {
+    $filter = $config_factory->getEditable($filter_id);
+    if ($filter->isNew()) {
+      continue;
+    }
+
+    $value = $filter->get('filters.filter_html.settings.allowed_html');
+    $replaced = str_replace('<iframe frameborder height scrolling src width>', '<iframe frameborder height scrolling src width title>', $value);
```

Comment:
Is this going to be brittle if anyone has changed the attributes allowed for an iFrame? Should it be a regex on the value that captures `<iframe (.*)>` and updates to `<iframe $1 title>`?

Reply:
it won't break but it will hide whatever is not allowed. It just adds a `display:none` in style to hide that field. Also if we put the wild card in, it will allow all the iframe fields to show up, which the content team does not want to have. For example, this is how the iframe window looks with all the fields 
![Screen Shot 2022-05-09 at 5 04 31 pm](https://user-images.githubusercontent.com/20810541/167357615-043cfe8e-fc72-4656-b116-7e189de98ae7.png)